extras/applications/FaceAndEyesDetection/Face&EyesDetection.py

```diff
@@ -0,0 +1,52 @@
+'''
```

Comment:
Rename this file as `extras/FaceAndEyesDetection/main.py`. Don't have `&` in the filename because it requires quoting in bash.